### ISSUE TITLE
SCORM XML analysis fails when item has no identifierref

### ISSUE THREAD

**Issue by graemechester** [numbas/blackboard-scorm-analysis#2]
Blackboard generated a manifest file containing the following:-

```
-<item identifier="itm00011">
<title>placeholder/staff-info</title>
+<item identifier="itm00012" identifierref="res00025">
</item>
```

Analysis failed when file None.dat was not found, since itm00011 has no resource file associated.
Workaround was possible by deleting the lines from the manifest.

**Comment by christianp** [comment]
Could you email your export to my ncl.ac.uk address please, so I have something to test against?

**Comment by graemechester** [comment]
Yes, sure. I didn't want to upload here as student data may be in the file.

**Comment by christianp** [comment]
Fixed by f616f45. I wonder why none of my exports had that item! Maybe there's been an update to Blackboard.

**Comment by graemechester** [comment]
Excellent, thanks Christian.

Kind regards,

## Graeme Chester

Dr. E.G. Chester, School of EEE, Newcastle University
Tel: 0191 208 7325 Email: graeme.chester@ncl.ac.ukmailto:graeme.chester@ncl.ac.uk
